src/vue/backend/src/templates/GeneralSettings.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+    <idx-block className="general-settings">
+        <idx-form-group>
+            <idx-form-label for="website-wrapper"><strong>Name Your Global Website Wrapper</strong><br> Wrappers set the overall styling of your IDX Broker pages, some words about maintaining a consistent design between WordPress and IDX Broker.</idx-form-label>
+            <idx-form-input
+                type="text"
+                id="website-wrapper"
+                :value="wrapperName"
+                @input="generalSettingsStateChange({ key: 'wrapperName', value: $event })"
```

Comment:
This seems like it could send a lot of unnecessary updates, lets chat about debouncing the input